models/product.model.lkml

```diff
@@ -3,6 +3,23 @@ connection: "snowflake"
 # Limit include only to the ones really needed by this explore
 include: "/views/marts/product/*.view.lkml"
 
+explore: fct_nps_feedback {
+  label: "NPS Feedback"
+  group_label: "[New] NPS Feedback"
```

Comment:
Perhaps use `[New] NPS` so that score can also be under the same group?

Reply:
Updated